### PR TITLE
Use updated CQM:Measure model

### PR DESCRIPTION
**Pull request by dczulada**
Pull requests into cqm-parsers require the following. Submitter and reviewer should :white_check_mark: when done. For items that are not-applicable, note it's not-applicable ("N/A") and :white_check_mark:.

**Submitter:**
- [ ] This pull request describes why these changes were made.
- [ ] Internal ticket for this PR:
- [ ] Internal ticket links to this PR
- [ ] Code diff has been done and been reviewed
- [ ] Tests are included and test edge cases
- [ ] Tests have been run locally and pass

**Reviewer 1:**

Name:
- [ ] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [ ] The tests appropriately test the new code, including edge cases
- [ ] You have tried to break the code

**Reviewer 2:**

Name:
- [ ] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [ ] The tests appropriately test the new code, including edge cases
- [ ] You have tried to break the code

### CODE REVIEW

File: lib/qrda-export/helper/aggregate_object_helper.rb [hunk 1]
```diff
@@ -98,30 +98,27 @@ def is_cv?
       class AggregateCount
         attr_accessor :measure_id,  :populations, :population_groups
 
+        ALL_POPULATION_CODES = %w[IPP DENOM NUMER NUMEX DENEX DENEXCEP MSRPOPL MSRPOPLEX OBSERV].freeze
```
Comment (okeefm) [inline lib/qrda-export/helper/aggregate_object_helper.rb]: We use these in Cypress too, in at least one place. Does it make sense to move these into a shared library? Maybe a constant on `Measure`?

Comment (losborne) [reply lib/qrda-export/helper/aggregate_object_helper.rb]: I believe they are already on `Measure` https://github.com/projecttacoma/cqm-models/blob/40b73f19f769415ccfd81c5161b856b64a3e3081/app/models/cqm/measure.rb#L25

Comment (dczulada) [reply lib/qrda-export/helper/aggregate_object_helper.rb]: Yeah, it has STRAT removed...since it isn't really a population.

Comment (okeefm) [reply lib/qrda-export/helper/aggregate_object_helper.rb]: I'm not sure how I feel about having the same constant name, with different values. Is there a reason for STRAT being in the `Measure` constant/should we remove it there?

Comment (dczulada) [reply lib/qrda-export/helper/aggregate_object_helper.rb]: This is no longer specified as a constant.
